README.md

```diff
@@ -475,12 +477,84 @@ specifically:
 | flight_windVelocity                 | Integer 16 |                             |
 
 
+***flight_type** is used to determine the type of the flight entity:
+
+    "Flight"			= 0,
+    "Positioning"		= 1,
+    "Non Flying"		= 2,
+    "Simulator"			= 3,
+    "Airport Reserve"	= 4,
+    "Airport Standby"	= 5,
+    "Home Reserve"		= 6,
+    "Home Standby"		= 7
+
+*Note: For any time field, if no night times are sent then it is assumed that the times are all "day". If `flight_night` is sent with the entity then the individual night times for other fields (PIC, XC, Dual, etc.) will auto fill. In most cases there isn't a need to send the individual night times.*
```

Comment:
If Out an In times are sent that would have Night time in it (with appropriate To and From fields), would Night ever auto-populate as well? (Or perhaps there is a modifier to the import that would allow this/LT calculates Total Time from Out/In as default?)